spec/features/todo_list_crud_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+feature "Todo list", :type => :feature do
+
+  scenario "create to-do list" do
+    sign_in("tester@example.tld", "test-password")
+    click_link "Lists"
+
+    create_new_list("New test list")
+  end
+
+  scenario "edit to-do list" do
+    sign_in("tester@example.tld", "test-password")
+    click_link "Lists"
+
+    create_new_list("New test list")
```

Comment:
The edit and delete scenarios don't need to do the data setup through the UI (via `create_new_list(...)`) if that's not the functionality they're concerned with testing. For a more focused, faster test consider using FactoryGirl for the list creation in the edit and delete scenarios.